cmd/spiffe-aws-assume-role/cli/cli_test.go

```diff
@@ -0,0 +1,110 @@
+package cli
+
+import (
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/square/spiffe-aws-assume-role/internal/mocks"
+	"github.com/square/spiffe-aws-assume-role/internal/test"
+	"github.com/square/spiffe-aws-assume-role/pkg/credentials"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
```

Comment:
Ugh. So goimports is good about getting non-SDK imports out of the first block but other than that it's a bit lacking... I'll clean this up manually.